sdk/nodejs/src/eventQueue.ts

```diff
@@ -17,58 +17,63 @@ export const EventTypes: Record<string, string> = {
     ...AggregateEventTypes
 }
 
-type EventsMap = Record<string, Record<string, DVCRequestEvent>>
-type UserEventsMap = {
-    user: DVCPopulatedUser,
-    events: EventsMap
-}
-type AggregateUserEventMap = Record<string, UserEventsMap>
-
 type UserEventsBatchRecord = {
     user: DVCPopulatedUser,
     events: DVCRequestEvent[]
 }
-export type FlushPayload = UserEventsBatchRecord[]
-type UserEventQueue = Record<string, UserEventsBatchRecord>
+export type FlushPayload = {
+    payloadId: string,
+    eventCount: number,
+    records: UserEventsBatchRecord[]
```

Comment:
This should probably be renamed to `EventsBatchRecord` now that it doesn't exclusively contain user-associated events?